web/src/app/services/data-store/data-store.service.ts

```diff
@@ -66,6 +66,27 @@ export class DataStoreService {
       );
   }
 
+  /**
+   * Returns the raw survey object from the db. Used for debbuging only.
+   */
+  async loadRawSurvey(id: string) {
+    return (
+      await firstValueFrom(
+        this.db.collection(SURVEYS_COLLECTION_NAME).doc(id).get()
+      )
+    ).data();
+  }
+
+  /**
+   * Updates the raw survey object in the db. Used for debbuging only.
+   */
+  async saveRawSurvey(id: string, data: any) {
```

Comment:
nit: `data: any` <--- Could this be string?

Reply:
How about this?